README.md

```diff
@@ -53,6 +53,7 @@ With the parser set, your configuration can be configured as described in the [C
 
 Additional configuration options can be set in your ESLint configuration under the `parserOptions` key. Please note that the `ecmaFeatures` config property may still be required for ESLint to work properly with features not in ECMAScript 5 by default.
 
+- `requireConfigFile` (default `true`) can be set to `false` to allow babel-eslint to run on files that do not have a Babel configuration associated with them. This can be useful for linting files that are not transformed by Babel (such as configuration files), though using [glob-based configuration](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns) to use the default parser for these files is recommended. Note that when no configuration file is found that babel-eslint will not parse any experimental syntax.
```

Comment:
> Note that when no configuration file is found that babel-eslint will not parse any experimental syntax.

that -> then?

Or maybe better as:

> Note, when no configuration file is found, `babel-eslint` will not parse any experimental syntax.

Reply:
Also, maybe:

> This can be useful for linting files that are not transformed by Babel (such as configuration files), though we recommend using the default parser via [glob-based configuration](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns).

Reply:
Updated!